src/index.html

```diff
@@ -11,7 +11,168 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="box-container">
+        <legend>Personal information</legend>
+        <label 
+        class="form-label" 
+        for="surname">
+          "Surname:"
+          <input 
+            type="text" 
+            autocomplete="off" 
+            name="surname" 
+            minlength="2" 
+            maxlength="30" 
+            required
+          >
+        </label>
+        <br>
+        <label class="form-label" for="name">
+          "Name:"
+          <input 
+            type="text" 
+            autocomplete="off" 
+            name="name" 
+            minlength="2" 
+            maxlength="30" 
+            required
+          >
+        </label>
+        <br>
+        <label class="form-label" for="age">
+          "How old are you? "
+          <input 
+            type="number" 
+            value="12" 
+            min="1" 
+            max="100" 
+            name="age" 
+            required
+          >
+        </label>
+        <br>
+        <label class="form-label" for="dob">
+          "Full data of birth"
+          <input 
+            type="date" 
+            name="dob" 
```

Comment:
don't use abbreviations or shorthands in code - better make names longer, but descriptive and clear
```suggestion
            name="date-of-birth" 
```